dracut/30ignition/coreos-remove-initramfs-network.service

```diff
@@ -0,0 +1,17 @@
+# Clean up the initramfs networking on first boot
+# so the real network is being brought up
+
+[Unit]
+Description=Tear down initramfs networking
+DefaultDependencies=false
+After=ignition-files.service
+Before=initrd.target
+
+# Make sure if ExecStart= fails, the boot fails
+OnFailure=emergency.target
+OnFailureJobMode=isolate
+
+[Service]
+Type=oneshot
+ExecStart=/usr/sbin/coreos-remove-initramfs-network
```

Comment:
Nit, let's make this `/usr/libexec` so it's not in default `$PATH` as a bash completion hazard.

Reply:
It doesn't exist in the real root so that's not an issue right? We also do this for e.g.https://github.com/coreos/ignition-dracut/blob/master/dracut/30ignition/coreos-populate-var.service#L14